### PR TITLE
making a change for demo purposes.

### PR DESCRIPTION
**Pull request by jlconlin**
Let's try this out.

### CODE REVIEW

**Comment by brown170** [comment]
It worked!

**Review by jlconlin — COMMENTED** [review]
This is a bad idea.

File: Agendas/May2017.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Paris 75016<br />
     3. Define API for *writing* GND data
     4. Tools for format verification
     5. Define standard list of physics tests.
- - 2:45--3:30 **Guidelines for conducting SG43 Business** (Jeremy Conlin)
+ - 2:45--3:30 **Guidelines for conducting SG43 Business** (The Great Jeremy Conlin)
```
Comment (whaeck) [inline Agendas/May2017.md:22]: This is my comment on it.

**Review by whaeck — APPROVED** [review]
I have no issues with this change. It can be merged into the master.

**Pull request closed without merging**